R/group.R

```diff
@@ -48,23 +48,32 @@ group_components <- function(type = 'weak') {
 #' @describeIn group_graph Group densely connected nodes using [igraph::cluster_edge_betweenness()]
 #' @importFrom igraph membership cluster_edge_betweenness
 #' @export
-group_edge_betweenness <- function(weights = NULL, directed = TRUE) {
+group_edge_betweenness <- function(weights = NULL, directed = TRUE, n_communities = NULL) {
   expect_nodes()
   weights <- enquo(weights)
   weights <- eval_tidy(weights, .E())
   # NULL in weights is for once respected despite a weight attribute
-  group <- as.integer(membership(cluster_edge_betweenness(graph = .G(), weights = weights, directed = directed)))
+  clusters <- cluster_edge_betweenness(graph = .G(), weights = weights, directed = directed)
+  if (!is.null(n_communities)) {
+    #clusters <- igraph::cut_at(clusters, no = n_communities)
+    return(clusters %>% igraph::cut_at(no = n_communities) %>%  as.integer %>% desc_enumeration)
+  }
+  group <- as.integer(membership(clusters))
```

Comment:
```suggestion
  if (!is.null(n_communities)) {
    group <- cut_at(clusters, no = n_communities)
  } else {
    group <- membership(clusters)
  }
  group <- as.integer(group)
```

Reply:
I'd prefer something like the above for all of the implementations, along with a `@importFrom igraph cut_at` line in the docs in order to import the cut_at function in the namespace

Reply:
Ok perfect, I will apply these changes and the documentation.